### ISSUE TITLE
nicoulaj wrong cursor place

### ISSUE THREAD

**Issue by Fraterius** [sorin-ionescu/prezto#911]
Theme nicoulaj has cursor at wrong place (no space after prompt, which leeds to other display errors after typing anything or using autocomplete). It only happens if my TERM is set to rxvt-unicode-256color (which is the Terminal I'm using). On Sorin theme everything is working ok. If I change TERM to rxvt-256color (or any other common TERM, like screen-256color or xterm-256color) nicoulaj also works ok. Not sure if it's a bug in rxvt, terminfo, or the theme itself.

If anything else is needed fill free to ask me.

I think it's a small bug which can be fixed easily, but sadly I don't have knowledge for that. Thanks for the great theme, it's a pleasure to work in the shell with nicoulaj :).

**Comment by sorin-ionescu** [comment]
It's a problem with how your terminal interprets Unicode characters.

**Comment by Fraterius** [comment]
Ok, can You point me to how to debug this issue or any other way to solve it? Why there is no such problem with Sorin theme?

Thx in advance for any tips.

**Comment by Fraterius** [comment]
Bump.

Are You sure it's my terminal fault, on any other theme from prezto everything seems to be working allright. Only with nicoulaj the problem exists.

I'd be more than happy to debug the problem, as nicoulaj is my favourite theme, right now I've switched to sorin and it's working ok but I like nicoulaj minimalism.

Thx in advance for any help on the subject.

**Comment by sorin-ionescu** [comment]
Show me a screencast.

**Comment by Fraterius** [comment]
Here it goes:
1. http://i.imgur.com/HIIecsd.png (here should be space before the cursor)
2. http://i.imgur.com/0kgvwIZ.png (like above)
3. http://i.imgur.com/1OetntR.png (autocompletion with tab, command goes to right position, but cursor is now shifted into the command)
4. http://i.imgur.com/B7xk29m.png (shows what's going on during deletion)

The worst is 4. when characters are left after deletion cause the cursor was shifted.

I've checked that the same error occurs with this theme on RoxTerm.

Thanks for Your help, if there's anything more I can do just say it. Today I've added glyphs to the rxvt-unicode with gnu unifont font but it didn't changed the problem, though sorin theme looks great while working with git repos.
